tc/sgx/trusted_worker_manager/enclave/signup_enclave_kme.cpp

```diff
@@ -38,11 +38,11 @@
 
 #define KME_SIGNUP_EXT_DATA_SIZE 32
 
-void CreateReportDataKME(std::string& enclave_signing_key,
+static void CreateReportDataKME(std::string& enclave_signing_key,
     const uint8_t* ext_data,
     sgx_report_data_t* report_data);
 
-void CreateSignupReportDataKME(const uint8_t* ext_data,
+static void CreateSignupReportDataKME(const uint8_t* ext_data,
```

Comment:
Any specific purpose to switch to static?

Reply:
Since the function is local to the scope of the file, its made static.